cmake/compiler/gcc/target_arc.cmake

```diff
@@ -9,4 +9,4 @@ if(NOT DEFINED GCC_ARC_TUNED_CPU)
 endif()
 
 list(APPEND TOOLCHAIN_C_FLAGS -mcpu=${GCC_ARC_TUNED_CPU})
-
+list(APPEND TOOLCHAIN_LD_FLAGS  -mcpu=${GCC_M_CPU})
```

Comment:
It should be:

```suggestion
list(APPEND TOOLCHAIN_LD_FLAGS  -mcpu=${GCC_ARC_TUNED_CPU})
```

Reply:
Yup, must have missed the commit adding the target tuning bits.